components/Tabs/Tabs.js

```diff
@@ -1,45 +1,39 @@
-
 class TabItem {
   constructor(element) {
-    // attach dom element to object. Example in Tabs class
-  }
-
-  select() {
-    // should use classList
+    this.element = element; // attach dom element to object. Example in Tabs class
   }
 
-  deselect() {
+  toggle() {
     // should use classList
+    this.element.classList.toggle('Tabs__item-selected');
   }
 }
 
 class TabLink {
   constructor(element, parent) {
-    this.element;// attach dom element to object
-    this.tabs;// attach parent to object
-    this.tabItem;// assign this to the associated tab using the parent's "getTab" method by passing it the correct data
-    // reassign this.tabItem to be a new instance of TabItem, passing it this.tabItem
-    this.element.addEventListener('click', () => {
+    this.element = element; // attach dom element to object
+    this.tabs = parent; // attach parent to object
+    this.tabItem = this.tabs.getTab(this.element.dataset.tab); // assign this to the associated tab using the parent's "getTab" method by passing it the correct data
```

Comment:
Try to keep comment on the line above, so they don't wrap around/ are longer than 80 chars